client-tools/Shell/V8ShellFeature.cpp

```diff
@@ -143,6 +144,11 @@ void V8ShellFeature::collectOptions(std::shared_ptr<ProgramOptions> options) {
       "--javascript.gc-interval",
       "Request-based garbage collection interval (each n-th command).",
       new UInt64Parameter(&_gcInterval));
+
+  options->addOption("--javascript.execution-deadline",
+                     "deadline in seconds. Once reached, calls will throw. "
+                     "Http timeouts will be adjusted.",
```

Comment:
Either HTTP or http, please. It's not a normal english word.

Reply:
fixed.